.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Test JTReg
+
+on:
+  push:
+    branches-ignore:
+        - master
+
+jobs:
+
+  linux-x64:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout the source
+        uses: actions/checkout@v2
+        with:
+          path: jtreg
+          fetch-depth: 0
+
+      - name: build
+        working-directory: jtreg
+        shell: bash
+        run: sh make/build-all.sh ${JAVA_HOME_8_X64}
```

Comment:
needs to be `make/build.sh`  

`make/build-all.sh` is obsolete and should now be deleted.